src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/update-or-disable-policies-conditions.mdx

```diff
@@ -26,81 +26,59 @@ import accountsConditionOff from 'images/accounts_icon_condition-off.webp'
+* To update the condition's [runbook URL](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/provide-runbook-instructions-alerts), select **Edit** in the ellipses (...) menu for the condition in the list.
 
-## Disable or delete conditions [#delete]
 
-<CollapserGroup>
-  <Collapser
-    id="condition-on-off"
-    title="Disable or re-enable a condition"
-  >
-    You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
+## Enable/disable a condition [#disable-conditions]
 
-    1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert Conditions**. Then, from the list of **Alert conditions**, utilize the toggle to enable or disable the condition.
-    2. Click the **On/Off** switch to toggle it.
+You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
 
-       <Callout variant="tip">
-         For infrastructure conditions, use the checkbox at the bottom of the condition edit screen, then save the condition.
-       </Callout>
+1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert Conditions**. Then, from the list of **Alert conditions**, utilize the toggle to enable or disable the condition.
+2. Click the **On/Off** switch to toggle it.
 
-       If you [copy a condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/copy-alert-conditions), it is automatically saved in the new policy as disabled (**Off**), even if the condition was enabled (**On**) in the original policy.
-  </Collapser>
+    <Callout variant="tip">
+      For infrastructure conditions, use the checkbox at the bottom of the condition edit screen, then save the condition.
+    </Callout>
 
-  <Collapser
-    id="delete-condition"
-    title="Delete conditions"
-  >
-    If a policy has multiple conditions, you can delete any or all of them, and the remaining conditions for the policy will continue to apply. To turn a condition off but keep it with the policy, [disable](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#condition-on-off) it. To delete one or more conditions:
+If you [copy a condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/copy-alert-conditions), it is automatically saved in the new policy as disabled (**Off**), even if the condition was enabled (**On**) in the original policy.
```

Comment:
Went through this to do some Vale fixes and came across "was enabled" as passive. Don't think we can really fix it without making the line more confusing but can we not remove the last bit entirely? It seems covered by the proceeding "saved in the new policy as disabled" part.